src/main/java/org/entando/kubernetes/config/AppConfiguration.java

```diff
@@ -103,4 +104,9 @@ public Map<ReportableRemoteHandler, AnalysisReportFunction> analysisReportStrate
                 kubernetesServiceClient::getAnalysisReport
         );
     }
+
+    @Bean
+    public RestTemplate simpleRestTemplate() {
+        return new RestTemplate();
```

Comment:
Better doing a test with self-signed certificates. Do we already have some config for that in the project?

Reply:
I'm thinking about that but I'm not sure that TLS could impact our approach. We are acting only on the path, not on the protocol, am I missing something?

Reply:
We use the ingress only when communicating with k8s-service (because it can be outside of the namespace). To communicate with de-app we leverage the service, so the communication should be kept unencrypted. I'm trying to test it manually